README.md

```diff
@@ -1,64 +1,102 @@
 # Quix
+
+> ## Disclaimer
+> :exclamation: Please note **that this is a beta version** of Quix which is still undergoing final testing before its official release.
+
 [![Build Status](https://travis-ci.com/wix-incubator/quix.svg?branch=master)](https://travis-ci.com/wix-incubator/quix)
-<br />
-Quix is a Web IDE for Presto.
 
-It's your single point of truth and a shared space for your company's BI insights, with quick turnaround, interactive visual perspectives, and mixed data sources.<br />
+Quix is a robust easy-to-use business intelligence & data analytics tool built to handle both quick insights and massive resource-demanding queries. It features data visualization and export, user management, and organizes queries in notebooks for easy access and sharing across teams.
+
+A web-based IDE for [Presto](https://github.com/prestodb/presto), Quix is your single point of truth and a shared space for your company's BI insights, with quick turnaround, interactive visual perspectives, and mixed data sources.
+
+![](docs/flow.gif)
+
+## What can I do with it?
+
+Quix is flexible enough to be used in an array of applications on any scale. It can serve as a:
 
-## Disclaimer
-Please note that this is a beta version of Quix which is still undergoing final testing before its official release.
+- Shared space for your company's BI insights
+- Single point of truth for all team members on public or proprietary data
+- Report generator using built-in charting tools
+- A handy tool to store, share, download, and embed live data from reports into any website (internal or public)
```

Comment:
The live version of Quix (the non-OSS one) I was able to use and play around with has a feature that allows to embed a note query result.

![image](https://user-images.githubusercontent.com/4682930/57194087-2b3a9280-6f4b-11e9-9034-65df1c476e95.png)

The Quix OSS [Google Doc](https://docs.google.com/document/d/1X14-c9mLNJyySKaZbFKZmbidwL0gDsChvL5fh3SU0Mw/edit) also mentioned sharing as a feature.

Thus I expected this would be built in to the public version of Quix OSS.

@antonpod should I remove any mentions of sharing capabilities for now? Are there any plans to add the sharing feature in the future at all?

Reply:
This feature is missing at the moment in OSS version, but we will add it sooner or later.

Reply:
In any case, I wouldn't market it as one of the main features, as it has some gotchas.
For example, to see the embedded note, you need to be logged into Quix.

Reply:
Oh, interesting. Okay, understood, thank you!